Makefile

```diff
@@ -34,9 +34,7 @@ default: release
 all: format build test
 
 pre-clippy:
-	@if [ "`cat clippy-version`" != "`cargo clippy --version || echo 0`" ]; then\
-		cargo install clippy --version `cat clippy-version` --force;\
-	fi
+	@rustup component add clippy-preview
```

Comment:
do we need to care the previous clippy like format does https://github.com/Hoverbear/tikv/blob/feb1338c78e130b9ec0f10ea30a08479f003e0c5/Makefile#L91

Reply:
I tried a few things and it seemed to work ok. Perhaps we can remove this from the format. I will experiment.